requirements/base.in

```diff
@@ -2,7 +2,7 @@
 
 -c constraints.txt
 
-Django>=1.11              # Web application framework
+Django                    # Web application framework
```

Comment:
I think that the minimum constraint is added in order to report to the dependencies in setup.py which is the minimum supported version of django and this is a library so I think that that should be added in here. But anyway I am going to approve this to see what are the thoughts of @jmbowman about that

Reply:
@jmbowman what do you think?

Reply:
While the lower boundary doesn't hurt, I'm not sure we really need it either.  I doubt if many people outside the Open edX ecosystem are using this package, and even fewer would be trying to do so with a version of Django so old as to be unsupported upstream.

Reply:
we've already constrained Django in `constraints.txt` to remain <2.3, should we be specifying here?